universal-application-tool-0.0.1/app/assets/javascripts/main.ts

```diff
@@ -40,6 +40,12 @@ function maybeHideElement(e: Event, id: string, parentId: string) {
   }
 }
 
+/** In admin program block edit form - enabling submit button when form is changed */
+function enableButton(event: Event) {

```

Comment:
(blocking) the name `enableButton` seems very generic, but this enables a very specific button. Please make the functionality match the generality of the name OR make the name match the specificity of the functionality.

Reply:
Resolved, renamed function for simplicity.